tests/stockfish/test_models.py

```diff
@@ -270,3 +270,33 @@ def test_get_parameters(self):
         arg1 = s1.get_parameters()
         arg2 = s2.get_parameters()
         assert arg1 != arg2
+
+    def test_get_top_moves(self):
+        stockfish = Stockfish(depth = 15, parameters = {"MultiPV": 3})
+        stockfish.set_fen_position("b5k1/1q3p1p/6p1/8/8/2Q3P1/1B3P1P/6K1 w - - 0 1")
+        top_moves = stockfish.get_top_moves(3)
+        assert ((top_moves[0]["Move"] == "c3g7" and top_moves[1]["Move"] == "c3h8") or
+               (top_moves[0]["Move"] == "c3h8" and top_moves[1]["Move"] == "c3g7"))
+        assert top_moves[2]["Move"] == "f2f3" or top_moves[2]["Move"] == "g1f1"
+        assert top_moves[0]["Centipawn"] == None and top_moves[0]["Mate"] == 1
+        assert top_moves[1]["Centipawn"] == None and top_moves[1]["Mate"] == 1
+        assert top_moves[2]["Centipawn"] == 0 and top_moves[2]["Mate"] == None
+    
+    def test_get_top_moves_mate(self):
+        stockfish = Stockfish(depth = 10, parameters = {"MultiPV": 3})
+        stockfish.set_fen_position("8/8/8/8/8/6k1/8/3r2K1 w - - 0 1")
+        assert stockfish.get_top_moves(2) == []
```

Comment:
It seems to me better to do assert for full top_moves

e.g. `assert topmoves == ...`

Reply:
Thanks for the feedback, I've updated my branch. For doing asserts on top_moves, is the code I added on line 284 of test-models what you had in mind?